processor.py

```diff
@@ -5,11 +5,22 @@
 # Defining main function
 def main():
     print("Hello World")
-    print_handling()
-    print_input()
-    print_storage()
+
+    storage = dataMaintainer()
+    handler = dataHandler()
+    reciever = dataReceiver()
+
+
+    handler.print_handling()
+    reciever.print_input()
+    storage.print_storage()
+
+    #print_handling()
```

Comment:
We will need to delete this commented out code to preserve the hygiene of our repo.